bindings/rust/evmc-vm/src/lib.rs

```diff
@@ -2,6 +2,195 @@ pub extern crate evmc_sys;
 pub use evmc_sys as ffi;
 
 // TODO: Add helpers for host interface
-// TODO: Add convenient helpers for evmc_result
 // TODO: Add convenient helpers for evmc_execute
 // TODO: Add a derive macro here for creating evmc_create
+
+/// EVMC result structure.
+pub struct ExecutionResult {
+    status: ffi::evmc_status_code,
+    gas: i64,
```

Comment:
I think `status_code` and `gas_left` is better to reduce deviation from the underlying structure.